### PR TITLE
Fix for flaky tests

### PR DESCRIPTION
**Pull request by donat**

### CODE REVIEW

File: org.eclipse.buildship.ui.test/src/main/groovy/org/eclipse/buildship/ui/launch/ClasspathSeparationTest.groovy [hunk 1]
```diff
@@ -245,6 +245,7 @@ class ClasspathSeparationTest extends SwtBotSpecification {
     private void launchAndWait(ILaunchConfiguration configuration) {
         ILaunch launch = configuration.launch(ILaunchManager.RUN_MODE, new NullProgressMonitor(), true)
         waitFor { launch.terminated }
+        waitFor { !consoles.activeConsoleContent.isEmpty() }
```
Comment (oehme) [inline org.eclipse.buildship.ui.test/src/main/groovy/org/eclipse/buildship/ui/launch/ClasspathSeparationTest.groovy]: Why do we need this?

Comment (donat) [reply org.eclipse.buildship.ui.test/src/main/groovy/org/eclipse/buildship/ui/launch/ClasspathSeparationTest.groovy]: I described the change in the commit message.

Comment (oehme) [reply org.eclipse.buildship.ui.test/src/main/groovy/org/eclipse/buildship/ui/launch/ClasspathSeparationTest.groovy]: Sorry, I failed at my own game :)

Comment (donat) [reply org.eclipse.buildship.ui.test/src/main/groovy/org/eclipse/buildship/ui/launch/ClasspathSeparationTest.groovy]: No, I am the one who usually omits the elaboration from the commit message.

**Pull request merged**